packages/schema/lib/schemas/ReadBulkSchema.js

```diff
@@ -0,0 +1,69 @@
+'use strict';
+
+const makeSchema = require('../utils/makeSchema');
+
+const BasicDisplaySchema = require('./BasicDisplaySchema');
+const BasicActionOperationSchema = require('./BasicActionOperationSchema');
+const KeySchema = require('./KeySchema');
+
+module.exports = makeSchema(
+  {
+    id: '/ReadBulkSchema',
+    description: 'How will Zapier fetch resources from your application?',
+    type: 'object',
+    required: ['key', 'noun', 'display', 'operation'],
+    examples: [
+      {
+        key: 'recipes',
```

Comment:
@xavdid @eliangcs looking at this again, I did `recipes` as the noun. Does it make sense to have the noun be plural (like I have it), or should I go with singular? Sorry, not super familiar w the nouns/schema...